legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -59,6 +59,7 @@ def _get_businesses_eligible_query():
                                   Batch.batch_type == Batch.BatchType.INVOLUNTARY_DISSOLUTION)
 
         query = db.session.query(Business).\
+            filter(not_(Business.admin_freeze.is_(True))).\
```

Comment:
This can be simplified to use `Business.admin_freeze.is_(False)`.

Reply:
I checked the db, there are some entries, actually a lot, with `null` value in the admin_freeze column. Should I ignore these?
![image](https://github.com/bcgov/lear/assets/169101247/797984e5-425d-4062-a8b7-35e7075a1226)

Reply:
Yep, should be good to keep your change. Ignore my comment.